packages/web/src/scss/components/UNSTABLE_PartnerLogo/_UNSTABLE_PartnerLogo.scss

```diff
@@ -0,0 +1,15 @@
+@use 'theme' as theme;
+@use 'tools';
+
+.UNSTABLE_PartnerLogo {
+    width: fit-content;
```

Comment:
We discussed this together using the example in the storybook. Without this setting, it stretches to the width of the entire parent element.

Reply:
And we know why. 💡 

I would rather choose this approach to prevent stretching → 
https://github.com/lmc-eu/spirit-design-system/pull/1532#discussion_r1662636937

Reply:
There is a question whether the width of the container should be adapted to the width of the logo or whether it should be stretched to the width of the parent element. This solution ensures that the width of the container fits the width of the logo - other solutions don't work for me, including the one you suggested 🤷‍♂️

Reply:
I'm able to replace the width with the display property, but I can't get the height to correctly fit if I send a significantly larger image there. It only works for me with`fit-content` 🤷‍♂️

Reply:
That's weird, it wouldn't. 🤔
But let it be for now and we can refactor it later…